docs/dev.rst

```diff
@@ -57,37 +57,20 @@ Managing dependencies
 Python dependencies
 -------------------
 
-Python dependencies for all parts of Socorro are split between two files:
+Python dependencies are maintained in the ``requirements.in`` file and
+"compiled" with dependencies of dependencies in the ``requirements.txt`` file.
 
-1. ``requirements/default.txt``, containing dependencies that Socorro uses
-   directly.
-2. ``requirements/constraints.txt``, containing dependencies required by the
-   dependencies in ``default.txt`` that Socorro does not use directly.
+To add a new dependency, add it to the file and then do::
 
-Dependencies in both files must be pinned and hashed. Use
-`hashin <https://pypi.python.org/pypi/hashin>`_.
-
-For example, to add ``foobar`` version 5::
-
-  $ hashin -r requirements/default.txt foobar==5
-
-If ``foobar`` has any dependencies that would also be installed, you must add
-them to the constraints file::
-
-  $ hashin -r requirements/constraints.txt bazzbiff==4.0
+   $ make shell
+   $ pip-compile --generate-hashes requirements.in
```

Comment:
@relud So one funny thing about this is that I need the Tecken Docker image to be built in order to run this inside the image. Otherwise I don't have the system dependencies installed to run `pip-compile` without it kicking up errors.

Is that how you add new dependencies in ingestion-edge? If not, how do you do it?

Reply:
I start a `pip install pip-tools` in a venv or the docker container i'm running

Reply:
:+1: